packages/web/src/executors/rollup/rollup.impl.ts

```diff
@@ -329,7 +329,11 @@ function updatePackageJson(
     /\.[jt]sx?$/,
     '.d.ts'
   );
-  packageJson.main = entryFileTmpl.replace('<%= extension %>', 'umd');
+  if (options.format.includes('umd')) {
+    packageJson.main = entryFileTmpl.replace('<%= extension %>', 'umd');
+  } else if (options.format.includes('cjs')) {
+    packageJson.main = entryFileTmpl.replace('<%= extension %>', 'cjs');
+  }
```

Comment:
What happens if only `esm` is specified as a format? Such as https://unpkg.com/lodash-es@4.17.21/package.json. Should there be a condition for `esm` as well?

Reply:
That's a good callout.

While testing this I was having problems with the names of `index.cjs.js` and `index.esm.js` on React Native.
Something was triggering it to look for something like `index.ios.js`, `index.native.js` etc and it was just ignoring the `index.cjs.js` despite the main entry being an exact match. I think the `js` in `cjs` there was triggering it.


I played around using the `index.cjs` and `index.mjs` of modern JavaScript but that game me issues with JavaScript modules importing common js with Webpack 4 (The codebase I am working on is a bit older)


What I have working locally is `index-esm.js` for esm modules and `index.js` for common js.

I would love some feedback on a direction to go with this, maybe it is a simple as using `index.common.js` instead of `index.cjs.js`?